commit c7dcf594f4c52fa7e2cc76918c8aa9abb98e9625
Author: Joseph Myers <josmyers@redhat.com>
Date:   Fri Nov 8 17:08:09 2024 +0000

    Rename new tst-sem17 test to tst-sem18
    
    As noted by Adhemerval, we already have a tst-sem17 in nptl.
    
    Tested for x86_64.

Conflicts:
	sysdeps/pthread/Makefile (renamed test)

diff --git a/sysdeps/pthread/Makefile b/sysdeps/pthread/Makefile
index de8b66bc3b6bf6ad..43fcdf1182e735e1 100644
--- a/sysdeps/pthread/Makefile
+++ b/sysdeps/pthread/Makefile
@@ -128,7 +128,7 @@ tests += tst-cnd-basic tst-mtx-trylock tst-cnd-broadcast \
 	 tst-pthread_kill-exiting \
 	 tst-cancel30 \
   tst-spin5 \
-  tst-sem17 \
+  tst-sem18 \
   tst-sem19 \
   tst-join16 \
 	 # tests
diff --git a/sysdeps/pthread/tst-sem17.c b/sysdeps/pthread/tst-sem18.c
similarity index 92%
rename from sysdeps/pthread/tst-sem17.c
rename to sysdeps/pthread/tst-sem18.c
index c3f05d196f4ef17a..1be207bcbeeb56f1 100644
--- a/sysdeps/pthread/tst-sem17.c
+++ b/sysdeps/pthread/tst-sem18.c
@@ -24,9 +24,9 @@
 int
 do_test (void)
 {
-  sem_unlink ("/glibc-tst-sem17");
+  sem_unlink ("/glibc-tst-sem18");
   errno = 0;
-  sem_t *s = sem_open ("/glibc-tst-sem17", 0);
+  sem_t *s = sem_open ("/glibc-tst-sem18", 0);
   TEST_VERIFY (s == SEM_FAILED);
   TEST_COMPARE (errno, ENOENT);
   return 0;
